### PR TITLE
GLT-1044 allow only numbers for timesReceived & tubeNumber

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

File: miso-web/src/main/webapp/scripts/sample_hot.js [hunk 1]
```diff
@@ -987,6 +987,14 @@ Sample.hot = {
       }
     }
     
+    function validatePosReqdNumber (value, callback) {
+      if (Handsontable.helper.isNumeric(value) && value > 0) {
```
Comment (apmasell) [inline miso-web/src/main/webapp/scripts/sample_hot.js]: This can also be:
`callback(Handsontable.helper.isNumeric(value) && value > 0);`

**Comment by apmasell** [comment]
:+1:

File: miso-web/src/main/webapp/scripts/sample_hot.js (hunk 1 above)
Comment (callunity) [inline miso-web/src/main/webapp/scripts/sample_hot.js]: I'll implement this in a PR coming soon!

**Comment by djcooke** [comment]
:+1:

**Pull request merged**